blob/azureblob/azureblob_test.go

```diff
@@ -0,0 +1,58 @@
+package azureblob
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"testing"
+	
+	"github.com/google/go-cloud/blob/driver"
+	"github.com/google/go-cloud/blob/drivertest"
+	"github.com/google/go-cloud/internal/testing/setup"
+)
+
+const (
```

Comment:
Also, can you add some comments here similar to the ones in `s3blob_test.go` about how to provision the bucket/container? I have an Azure account now, but it's not obvious how to create a blob container.

Reply:
Comments added.